chapters/ch20.md

```diff
@@ -1,49 +1,57 @@
 # 20. Bias and Variance: The two big sources of error
 ->
-
+# 20. Độ chệch và phương sai: Hai nguồn lớn của lỗi
 
 Suppose your training, dev and test sets all come from the same distribution. Then you should always try to get more training data, since that can only improve performance, right?
 ->
-
+Giả sử khi huấn luyện, tập phát triển và tập kiểm tra có cùng phân phối. Khi đó bạn luôn nên cố gắng thu thập thêm dữ liệu, vì điều đó chỉ giúp cải thiện hiệu quả, đúng không? (cần dịch lại)
 
 Even though having more data can’t hurt, unfortunately it doesn’t always help as much as you might hope. It could be a waste of time to work on getting more data. So, how do you decide when to add data, and when not to bother?
 ->
+Mặc dù có thêm dữ liệu không thể giảm hiệu quả, thật không may điều này không luôn luôn mang lại hiệu quả nhiều như bạn mong đợi. Việc thu thập thêm dữ liệu có thể trở nên lãng phí thời gian. Vậy làm thể nào để quyết định khi nào nên thêm dữ liệu, khi nào không nên?
 
 
 There are two major sources of error in machine learning: bias and variance. Understanding them will help you decide whether adding data, as well as other tactics to improve performance, are a good use of time.
 ->
-
+Có hai nguồn chính dẫn đến lỗi trong machine learing: độ chệch và phương sai. Hiểu được chúng sẽ giúp bạn quyết định liệu rằng có đáng bỏ ra thời gian để thêm dữ liệu, cũng như các kỹ thuật khác để cải thiện hiệu quả của mô hình.
```

Comment:
"Hiểu được chúng sẽ giúp bạn quyết định liệu rằng có đáng bỏ ra thời gian để thêm dữ liệu, cũng như các kỹ thuật khác để cải thiện hiệu quả của mô hình." ==> "Hiểu được chúng sẽ giúp bạn quyết định liệu rằng khi nào thì nên bỏ thời gian để thêm dữ liệu hay là dành thời gian cho các kỹ thuật khác để cải thiện hiệu quả của mô hình."

Reply:
Theo mình hiểu thì cấu trúc của đoạn "whether ... are a good use of time" ý nói rằng "liệu rằng làm ... có phải cách sử dụng thời gian một cách hợp lý". 

Còn theo bạn thì sẽ thành "liệu rằng khi nào nên bỏ thời gian". Câu này của bạn cũng có một chút vấn đề về ngữ pháp, người ta chỉ nói "liệu rằng làm cái này cái kia có tốt không", chứ không dùng "liệu rằng làm cái này như thế nào" hay "liệu rằng khi nào làm cái này"

Reply:
em nghĩ câu này anh hiểu đúng rồi. Link dưới đây cũng confirm ý của anh. 

https://www.quora.com/What-does-it-mean-to-make-good-use-of-my-time